Index: log4cpp/src/Makefile.am
===================================================================
--- log4cpp/src/Makefile.am
+++ log4cpp/src/Makefile.am	2018-04-04 12:32:20.822060155 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES = -I$(top_srcdir)/include
 
-noinst_HEADERS = snprintf.c Localtime.hh
+noinst_HEADERS = Localtime.hh
 
 liblog4cpp_la_SOURCES = \
 	Appender.cpp \
Index: log4cpp/src/Makefile.in
===================================================================
--- log4cpp/src/Makefile.in
+++ log4cpp/src/Makefile.in	2018-04-04 12:32:46.982045443 +0200
@@ -297,7 +297,7 @@
 top_srcdir = @top_srcdir@
 lib_LTLIBRARIES = liblog4cpp.la
 INCLUDES = -I$(top_srcdir)/include
-noinst_HEADERS = snprintf.c Localtime.hh
+noinst_HEADERS = Localtime.hh
 liblog4cpp_la_SOURCES = Appender.cpp AppenderSkeleton.cpp \
 	AppendersFactory.cpp BufferingAppender.cpp FactoryParams.cpp \
 	LayoutsFactory.cpp LevelEvaluator.cpp Localtime.cpp \
